fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMultiTargetTest.java

```diff
@@ -948,4 +948,61 @@ void testForbidConversionOfActiveMultiTarget() throws InterruptedException {
         // happy indexes assertion
         assertReadable(indexes);
     }
+
+    @Test
+    void testForbidConversionOfActiveMultiTargetToMutual() throws InterruptedException {
```

Comment:
Does the "Active" matter, wouldn't this code forbid the conversion regardless.

Reply:
Not really. The mutual does not use the sync lock.